documentation/src/main/markdown/troubleshooting/solutions.md

```diff
@@ -193,3 +193,30 @@ The problem may be intermittent.
 There is no harm in leaving the directories behind in the short term but we recommend periodically removing them if the problem occurs frequently.
 Restarting Docker will force Docker to release the locks, and enable you to remove the directories.
 
+## Encoding Problems with PIP Requirements File
+
+### Symptom
+
+The requirements.txt files created using encoding systems other than UTF-8 cause certain unicode characters in the component names to be unreadable when inspected through the Detect. The system does not recognize these component entries, resulting in unmatched components. 
```

Comment:
Tiny tweak suggestion:

`requirements.txt` files created using encoding systems other than UTF-8 cause certain Unicode characters in the component names to be unreadable when inspected through [detect_product_short]. The system does not recognize these component entries, resulting in unmatched components.

Reply:
Fixed.